### PR TITLE
Make tests for helpers

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

File: config/.env.template
Comment (helendduncan) [inline config/.env.template:1]: How would you feel about leaving the env.template as is and renaming this something like env.actions - I know it's a pedantic thing but the installation instructions reference it and it might get confusing?

Comment (gedankenstuecke) [reply config/.env.template:1]: Great idea, just made a push for that!

File: server/apps/main/tests/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        # Load context data from JSON rather than VCR, as should be independent
+        # of using external fixtures. Basic JSON contains 4 experiences for testing
+        with open("server/apps/main/tests/fixtures/helper-context.json") as f:
+            self.context = json.load(f)
+        data = {"access_token": 'foo',
+        "refresh_token": 'bar',
+        "expires_in": 36000}
+        self.non_moderator_user = OpenHumansMember.create(oh_id=12345678,data=data)
+        self.moderator_user = OpenHumansMember.create(oh_id=23456789,data=data)
+        self.moderator_group = Group.objects.create(name='SecondGroup')
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply server/apps/main/tests/test_helpers.py]: I've made an edit to clarify the setup! But to also have it here: 

I wanted the "moderator" user to be in two groups at the same time to trigger the edge-case for the *is_moderator* function in that the function originally broke when the user is in >1 group. 

That's why I created the second group and added the user to that one as well. but inded it's a bit confusing that it was re-using the `self.moderator_group`. It worked as it should regardless, as it overwrote the `self.moderator_group` definition, but now it should be clearer!

[the remark this answers is not in the document]
Comment (helendduncan) [reply server/apps/main/tests/test_helpers.py]: Ah gotcha thank you!

**Review by helendduncan — APPROVED** [review]
